hyperstart.go

```diff
@@ -403,6 +403,50 @@ func (h *hyper) exec(pod *Pod, c Container, process Process, cmd Cmd) error {
 	return nil
 }
 
+func (h *hyper) addDrives(pod *Pod) error {
```

Comment:
Here as well, this should be a `Pod` private method for other hypervisor implementations to use the same logic.